packages/odyssey-react/src/components/TextInput/TextInput.theme.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  ColorBorder: theme.ColorBorderUi,
+  ColorBorderInvalid: theme.ColorBorderDangerBase,
+  ColorBorderDisabled: theme.ColorBorderDisabled,
+  ColorBackground: theme.ColorBackgroundBase,
+  ColorBackgroundDisabled: theme.ColorBackgroundDisabled,
+  ColorText: theme.ColorTextBody,
+  ColorTextDisabled: theme.ColorTextBody,
+  ColorTextPlaceholder: theme.ColorTextSub,
+  ColorFocusOutline: theme.ColorFocusPrimary,
+  ColorFocusOutlineInvalid: theme.ColorFocusDanger,
+  BorderWidth: theme.BorderWidthBase,
+  BorderStyle: theme.BorderStyleBase,
+  BorderRadius: theme.BorderRadiusBase,
+  PaddingBlock: theme.SpaceEmXs,
+  PaddingInline: theme.SpaceEmS,
+  MaxWidth: theme.FontLineLengthMax,
+  FontFamily: theme.FontFamilyBase,
+  FontSize: theme.FontSizeBody,
+  LineHeight: theme.FontLineHeightBase,
+  FocusOutlineWidth: theme.FocusOutlineWidthBase,
+  TransitionDuration: theme.TransitionDurationBase,
+  TransitionTiming: theme.TransitionTimingBase,
+  RtlBackgroundPositionRightOffset: theme.SpaceRemS,
+  SearchPaddingInlineStart: theme.SpaceRemXs,
+  IndicatorSize: "1.1487rem",
```

Comment:
`theme.FontSizeHeading5`? (Naming aside, I believe this is intended to be associated with the size one step up from the base of the scale.)

Reply:
On topic: I encountered one size that wasn't part of our current mapping in Text but decided against adding a new entry (kept it local for now)

Reply:
Yes, the previous implementation was using the scale value to go up one step. Mapping this value back to something intended for heading sizes feels more brittle than a magic number right now.

Reply:
Good call out @andrewberg-okta. @josesolano-okta and I can revisit this when we propose the new type tokens.

I think we'll want to find a standard for these Icon/Indicator sizes that doesn't attach them to something as specific as a Heading style.